ReactiveCocoa/UIKit/UIControl.swift

```diff
@@ -62,6 +62,11 @@ extension Reactive where Base: UIControl {
 		}
 	}
 
+	@available(*, unavailable, renamed: "controlEvents(_:)")
```

Comment:
I went back and forth on that, and thought I'd just put it here because the file doesn't exist for RAC and this is temporary (to be removed by 5.0). But on the other hand I can see how creating that file makes it more visible than "burying" it here alongside the renamed method.